### PR TITLE
add go get commands for all external databases

### PR DESCRIPTION
**Pull request by Umang01-hash**
**Description:**

-   Resolve issue #1180 

**Checklist:**

-   [x] I have formatted my code using  `goimport`  and  `golangci-lint`.
-   [x] All new code is covered by unit tests.
-   [x] This PR does not decrease the overall code coverage.
-   [x] I have reviewed the code comments and documentation for clarity.

**Thank you for your contribution!**

### CODE REVIEW

File: docs/advanced-guide/injecting-databases-drivers/page.md [hunk 1]
```diff
@@ -20,6 +20,13 @@ type Clickhouse interface {
 
 User's can easily inject a driver that supports this interface, this provides usability without
 compromising the extensibility to use multiple databases.
+
+Import the gofr's external driver for ClickHouse:
+
+```go
+go get gofr.dev/pkg/gofr/datasource/clickhouse
```
Comment (vipul-rawat) [inline docs/advanced-guide/injecting-databases-drivers/page.md:27]: Please mention @latest for version

File: docs/advanced-guide/injecting-databases-drivers/page.md [hunk 2]
```diff
@@ -104,6 +111,13 @@ type Mongo interface {
 
 User's can easily inject a driver that supports this interface, this provides usability without
 compromising the extensibility to use multiple databases.
+
+Import the gofr's external driver for MongoDB:
+
+```go
+go get gofr.dev/pkg/gofr/datasource/mongo
+```
```
Comment (ccoVeille) [inline docs/advanced-guide/injecting-databases-drivers/page.md:119]: The fenced block type is wring (here and all `go get` you add)


````suggestion
```shell
go get gofr.dev/pkg/gofr/datasource/mongo
```
````

**Review by ccoVeille — CHANGES_REQUESTED** [review]

File: docs/advanced-guide/injecting-databases-drivers/page.md (hunk 1 above)
Comment (Umang01-hash) [reply docs/advanced-guide/injecting-databases-drivers/page.md:27]: sure done

File: docs/advanced-guide/injecting-databases-drivers/page.md (hunk 2 above)
Comment (Umang01-hash) [reply docs/advanced-guide/injecting-databases-drivers/page.md]: Sure done

**Pull request merged**